lib/Config.ts

```diff
@@ -152,6 +158,11 @@ class Config {
         this.updateMacaroonPaths();
       }
 
+      if (props.limits) {
```

Comment:
Is this something we want to make configurable? That's not the impression I was under and it's not mentioned in the issue. @kilrau

Reply:
@sangaman correct I don't think we want this to be configurable for now.

Reply:
Wouldn't we want to remove this code then?